eve/bin/os_script_init.sh

```diff
@@ -41,8 +41,10 @@ then
 
   if [ "$dist_major" == "8" ]; then
     retry dnf -y install python2-devel python2-setuptools
+    python_binary=/bin/python2
   else
     retry yum -y install python-devel python-setuptools
+    python_binary=/bin/python
```

Comment:
I think you can use "python2" for all distribs, its available on all centos and ubuntu as far as i know